python/graphstorm/eval/evaluator.py

```diff
@@ -2064,7 +2184,13 @@ def best_iter_num(self):
     def val_perf_rank_list(self):
         raise RuntimeError("GSgnnMultiTaskEvaluator.val_perf_rank_list not supported")
 
-    def evaluate(self, val_results, test_results, total_iters):
+    def evaluate(
+        self,
+        val_results: Dict[str, Any],
+        test_results: Dict[str, Any],
+        total_iters: int,
+        **kwargs
```

Comment:
It seems kwargs are not used.

Reply:
Yes, the MT evaluator does not use the `kwargs` currently. I just added it here to have a consistent interface with the other evaluators, maybe we can remove it?